test/helpers/contract-helpers.ts

```diff
@@ -164,89 +165,165 @@ export const initializeContract = async () => {
+export const deposit = async (
+  ownerId: number,
+  ownerAddress: string,
+  operatorIds: number[],
+  amount: string,
+  cluster: any,
+) => {
   await DB.ssvToken.connect(DB.owners[ownerId]).approve(DB.ssvNetwork.contract.address, amount);
-  const depositedCluster = await trackGas(DB.ssvNetwork.contract.connect(DB.owners[ownerId]).deposit(
-    ownerAddress,
-    operatorIds,
-    amount,
-    cluster));
+  const depositedCluster = await trackGas(
+    DB.ssvNetwork.contract.connect(DB.owners[ownerId]).deposit(ownerAddress, operatorIds, amount, cluster),
+  );
   return depositedCluster.eventsByName.ClusterDeposited[0].args;
 };
 
 export const liquidate = async (ownerAddress: string, operatorIds: number[], cluster: any) => {
-  const liquidatedCluster = await trackGas(DB.ssvNetwork.contract.liquidate(
-    ownerAddress,
-    operatorIds,
-    cluster
-  ));
+  const liquidatedCluster = await trackGas(DB.ssvNetwork.contract.liquidate(ownerAddress, operatorIds, cluster));
   return liquidatedCluster.eventsByName.ClusterLiquidated[0].args;
 };
 
 export const removeValidator = async (ownerId: number, pk: string, operatorIds: number[], cluster: any) => {
-  const removedValidator = await trackGas(DB.ssvNetwork.contract.connect(DB.owners[ownerId]).removeValidator(
-    pk,
-    operatorIds,
-    cluster
-  ));
+  const removedValidator = await trackGas(
+    DB.ssvNetwork.contract.connect(DB.owners[ownerId]).removeValidator(pk, operatorIds, cluster),
+  );
   return removedValidator.eventsByName.ValidatorRemoved[0].args;
 };
 
 export const withdraw = async (ownerId: number, operatorIds: number[], amount: string, cluster: any) => {
-  const withdrawnCluster = await trackGas(DB.ssvNetwork.contract.connect(DB.owners[ownerId]).withdraw(
-    operatorIds,
-    amount,
-    cluster));
+  const withdrawnCluster = await trackGas(
+    DB.ssvNetwork.contract.connect(DB.owners[ownerId]).withdraw(operatorIds, amount, cluster),
+  );
 
   return withdrawnCluster.eventsByName.ClusterWithdrawn[0].args;
 };
 
 export const reactivate = async (ownerId: number, operatorIds: number[], amount: string, cluster: any) => {
   await DB.ssvToken.connect(DB.owners[ownerId]).approve(DB.ssvNetwork.contract.address, amount);
-  const reactivatedCluster = await trackGas(DB.ssvNetwork.contract.connect(DB.owners[ownerId]).reactivate(
-    operatorIds,
-    amount,
-    cluster));
+  const reactivatedCluster = await trackGas(
+    DB.ssvNetwork.contract.connect(DB.owners[ownerId]).reactivate(operatorIds, amount, cluster),
+  );
   return reactivatedCluster.eventsByName.ClusterReactivated[0].args;
 };
 
-export const registerValidators = async (ownerId: number, numberOfValidators: number, amount: string, operatorIds: number[], gasGroups?: GasGroup[]) => {
+export const registerValidatorsWithKeystores = async (
+  ownerId: number,
+  amount: string,
+  keystores: string[],
+  passwords: string[],
```

Comment:
remove `keystores` and `passwords`. It should receive the ID of the validator we want to register. Later the public/private key will be taken from the validatorsData JSON using this ID